test/prob/discrete_range/discrete_range_ccdf_log_test.hpp

```diff
@@ -0,0 +1,65 @@
+// Arguments: Ints, Ints, Ints
+#include <stan/math/prim.hpp>
+
+using stan::math::var;
+using std::vector;
+
+class AgradCcdfLogDiscreteRange : public AgradCcdfLogTest {
+ public:
+  void valid_values(vector<vector<double>>& parameters,
+                    vector<double>& ccdf_log) {
+    vector<double> param(3);
+
+    param[0] = 3;  // y
+    param[1] = 1;  // lower
+    param[2] = 5;  // upper
+    parameters.push_back(param);
+    ccdf_log.push_back(-0.916290731874154995573);  // expected ccdf_log
```

Comment:
Rather than magic numbers like this, the expected value should be `log(2.0 / 5)`.  It's easy to see that's the right ansewr because there are 2 elements strictly greater than y.

Same for others.